doc/read-the-docs-site/conf.py

```diff
@@ -82,7 +82,8 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = [
     'haddock', # Otherwise it tries to pick up the README.md's in the Haddock doc!
-    'README.md'
+    'README.md',
+    'stdspan.md'
```

Comment:
Otherwise we get:
```
Warning, treated as error:
       > /private/tmp/nix-build-read-the-docs-site.drv-0/source/srcspan.md:document isn't included in any toctree
```
Which makes me think: is `srcspan.md` supposed to be in the `read-the-docs-site` folder? 🤔

Reply:
probably not. I think this might have gone wrong when you rearranged things?

Reply:
I thought so too but I just went through the history and originally it was in the `doc` folder so... @bezirg what would be a better place for this file? Maybe `doc/notes`?

Reply:
yes